packages/skeleton/src/lib/components/ListBox/ListBox.svelte

```diff
@@ -17,6 +19,8 @@
 	// Props (styles) - Item Only
 	/** Provide classes to set the listbox item active background styles. */
 	export let active: CssClasses = 'variant-filled';
+	/** Provide classes to set the listbox item active disabled background styles */
+	export let activeDisabled: CssClasses = 'variant-ghost';
```

Comment:
Typically, we don't offer a "disabled-style" prop. 
Instead, we provide a default styling for disabled elements (* users can always override these styles). However, if we decide to introduce such a feature, let's consider renaming the prop and updating the corresponding comment for clarity.

Reply:
I've renamed the class and holding on removing if there is an intention in providing this feature, as you mentioned.

Reply:
This can be closed if you agree with https://github.com/skeletonlabs/skeleton/pull/2485#discussion_r1501894198